app/directives/meetings/admeetings.directive.js

```diff
@@ -1,7 +1,7 @@
 /**
- * (c) 2016 Tieto Finland Oy
- * Licensed under the MIT license.
- */
+* (c) 2016 Tieto Finland Oy
```

Comment:
Previous indentation is the more common way used in file header comment blocks: 
/**
_*
_*/
not 
/*
*
*/